e2e/tests/policy-forms.spec.ts

```
@@ -867,6 +835,217 @@ spec:
+  test('create form renders with required fields', { tag: '@smoke' }, async ({ page }) => {
+    await gotoPage(page, createPagePath(TEST_NAMESPACE, 'kuadrant.io~v1~RateLimitPolicy'));
+
+    await expect(page.getByRole('heading', { name: 'Create RateLimit Policy' })).toBeVisible({
+      timeout: 15_000,
+    });
+
+    // Form tab should be selected by default
+    await expect(page.getByRole('tab', { name: 'Form' })).toHaveAttribute(
+      'aria-selected',
+      'true',
+    );
```

ESLint

[error] Replace `⏎······'aria-selected',⏎······'true',⏎····` with `'aria-selected',·'true'`

(prettier/prettier)